Dockerfile-debian.template

```
@@ -49,7 +49,8 @@ RUN set -eux; \
 		! grep -q '/usr/share/locale' /etc/dpkg/dpkg.cfg.d/docker; \
 	fi; \
 	apt-get update; apt-get install -y --no-install-recommends locales; rm -rf /var/lib/apt/lists/*; \
-	localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+	echo en_US.UTF-8 UTF-8 >> /etc/locale.gen; \
+	locale-gen
```

Is there something we could run to verify that this actually worked afterwards?  Maybe something like `locale -a | grep en_US.utf8`?

```suggestion
	echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen; \
	locale-gen; \
	locale -a | grep 'en_US.utf8'
```